releasenotes/notes/preset-pass-manager-approximation-degree-503198b1974adf79.yaml

```
@@ -0,0 +1,12 @@
+---
+upgrade:
+  - |
+    The implicit use of ``approximation_degree!=1.0`` by default in the
+    :func:`~.generate_preset_pass_manager` function at ``optimization_level=3`` has been
```

This is actually potentially called at any optimization level, it's just that in < level 3 we only use if there are explicitly `UnitaryGate`s in the circuit which need to be synthesized.

```suggestion
    :func:`~.generate_preset_pass_manager` function has been
```